src/backend/server.js

```diff
@@ -263,7 +279,8 @@ app.get("/start/:chatId/:userId", async (req, res) => {
     if (keys.isBeta && !whitelist) {
       await db.addWhitelistGroupId(chatId);
     }
-    await db.createUser(userId, chatId);
+    const modifiedUserId = user * keys.hiddenLargeConstant;
```

Comment:
Modification in the userId should be done in the telegram side. Afterwards you can put a check (like in line 218) here before you create a new user if the user isn't registered yet

Reply:
Shifted multiplication to telegram.js.